client/src/app/queries/migration-waves.ts

```diff
@@ -27,11 +29,14 @@ export const useCreateMigrationWaveMutation = (
 };
 
 export const useFetchMigrationWaves = () => {
+  const queryClient = useQueryClient();
   const { isLoading, error, refetch, data } = useQuery({
     queryKey: [MigrationWavesQueryKey],
     queryFn: getMigrationWaves,
     refetchInterval: 5000,
     onError: (error) => console.log("error, ", error),
+    onSuccess: () => queryClient.invalidateQueries([TicketsQueryKey]),
+    select: (waves) => getWavesWithStatus(queryClient, waves),
```

Comment:
To avoid systematically fetching all status data can't we create a separate fetch query ?
This will avoid to overload migrationwave data with the status data when only the migration data itself is only needed.

Reply:
Not sure that we use the waves anywhere else? Maybe if we see a performance issue we could address this.